angular.json

```diff
@@ -133,7 +139,6 @@
               "path": "./templates/webpack/webpack.custom.ts",
               "replaceDuplicatePlugins": true
             },
-            "showCircularDependencies": false,
```

Comment:
Yes, this is not related but removes an build warning and since I touched the file ...
I could save this for an upcoming Angular/dependencies update PR where it might fit better to keep this Commit clean?

Reply:
I removed it and it might return later. ;)

Reply:
Just add a new one-line PR and merge it 😅